package-lock.json

```diff
@@ -7,6 +7,7 @@
     "": {
       "name": "@onfido/castor-icons",
       "version": "2.0.0",
+      "hasInstallScript": true,
```

Comment:
Does this mean people that do `npm i @onfido/castor-icons` won't have `postinstall` run automatically, only contributors when installing this repo?

Reply:
TBH I do not know what this exactly means...

But no, we do not distribute `package-lock.json` nor we leave "scripts" in `package.json`.

For the record, if we would be distributing same `package.json` and package _is_ public, an additional library `pinst` is then recommended: https://typicode.github.io/husky/#/?id=install